nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/FusedBatchNorm.java

```diff
@@ -0,0 +1,47 @@
+package org.nd4j.linalg.api.ops.custom;
```

Comment:
All new ops should have Konduit K.K. copyright header

Reply:
Done